tokenserver/scripts/purge_old_records.py

```diff
@@ -65,6 +66,12 @@ def purge_old_records(config_file, grace_period=-1, max_per_loop=10,
                     "offset": offset,
                 }
                 rows = list(backend.get_old_user_records(service, **kwds))
+                if not rows:
+                    logger.info("No more data for %s", service)
+                    break
+                if [service].extend(rows) == previous_list:
```

Comment:
What's `[service]` doing? `extend` going to add on inplace, always returning `None`.

`service` here is a value on the user record but it's more of a foreign key to the service table. I don't think we actually use it, it's legacy from when tokenserver was built to support multiple services.

At one point we had a 'sync-1.1' and 'sync-1.5' protocols to support but I don't think tokenserver even used this feature to support that. Long story short, consider `service` to be unrelated, separated buckets of users on potentially different services.